pyanaconda/ui/gui/spokes/custom_storage.py

```diff
@@ -130,6 +127,14 @@ def __init__(self, data, storage, payload):
         self._boot_loader = STORAGE.get_proxy(BOOTLOADER)
         self._disk_selection = STORAGE.get_proxy(DISK_SELECTION)
 
+    @property
+    def category(self):
+        return SystemCategory
+
+    @property
+    def title(self):
+        return _("MANUAL PARTITIONING")
```

Comment:
This looks like a bug. Please, replace the upper-cased string with a capitalized string.

Reply:
Will do - I wonder if the custom partitioning spoke title is ever displayed anywhere given that it is only ever accessed via the Storage spoke and never directly presented on the a hub, where the title string can be seen.

That could explain why this might have been overlooked in the past.